src/components/Logo/Logo.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import cx from 'classnames';
+
+type LogoProps = {
+  title?: string;
+  fill?: string;
+  className?: string;
+  data?: {
+    viewBox?: string;
+    fillRule?: 'nonzero' | 'evenodd' | 'inherit';
+    path?: string;
+  };
+};
+
+export const Logo = ({
+  title = '',
```

Comment:
Would remove default assignments (`= ''`) as they're unnecessary.

Reply:
I've made all the changes as requested.

Reply:
Cool will review/approve once pushed

Reply:
@DipakShrestha18 please can you try pushing your changes again. I can still see the default assignments

Reply:
done.